managed/services/supervisord/supervisord.go

```diff
@@ -445,13 +446,20 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 
 	s.addPostgresParams(templateParams)
 
+	templateParams["PMMServerAddress"] = defaultHostAddress
```

Comment:
Minor: I think it's also makes sense to rename this template param to `PMMServerHost`  or something like this.